### DISCUSSION TITLE
Why does the extracted-content field only return a maximum of 8 pieces of data after structuring the large model data

### DISCUSSION THREAD

**Discussion by sz0811** [unclecode/crawl4ai#530] [Forums - Q&A]
换了不同的大模型调用，生成的markdown数据是准确的，再由大模型生成结构化的数据后，数据缺失，最多只返回8条数据是为何？哪里可以设置吗？

**Comment by sz0811** [comment]
After using different large model calls, the generated markdown data is accurate. However, when the large model generates structured data, the data is missing and only a maximum of 8 pieces of data are returned. What is the reason for this? Where can I set it up?

**Comment by unclecode** [comment]
@sz0811 This is interesting, There's no limit in a library unless the large language model you used detected only the eight pieces of data. Please share the code snippet and the language model you are using. I tried on my end and will update you.
